styles.css

```diff
@@ -309,11 +305,11 @@ h4 {
 .close-button {
   background-image: url('x-mark-3-512.png');
   background-size: 100%;
+  background-repeat: no-repeat;
```

Comment:
Is this for when the x button would repeat itself when the screen would get too big?

Reply:
so when the screen shrinks to much it doesn't repeat the  background imgage we used

Reply:
 ✨